app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/createPasswordProtectedGuestLink/CreatePasswordProtectedGuestLinkScreen.kt

```diff
@@ -0,0 +1,260 @@
+                    .padding(internalPadding)
+                    .padding(
+                        start = dimensions().spacing16x,
+                        end = dimensions().spacing16x,
+                        bottom = dimensions().spacing16x,
+                        top = dimensions().spacing16x,
+                    )
+                    .weight(1F)
+                    .fillMaxSize()
+            ) {
+
+                item {
+                    Text(
+                        text = stringResource(
+                            id = R.string.conversation_options_create_password_protected_guest_link_discrption
+                        ),
+                        style = MaterialTheme.wireTypography.body02.copy(fontWeight = FontWeight.Normal)
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+                    Text(
+                        text = stringResource(
+                            id = R.string.conversation_options_create_password_protected_guest_link_discrption_2
+                        ),
+                        style = MaterialTheme.wireTypography.body02
+                    )
+                    Spacer(modifier = Modifier.height(24.dp))
+                }
+
+                item {
+                    GeneratePasswordButton(
+                        onClick = viewModel::onGenerateRandomPassword
+                    )
+                    Spacer(modifier = Modifier.height(24.dp))
+                }
+
+                item {
+                    WirePasswordTextField(
+                        labelText = stringResource(
+                            id = R.string.conversation_options_create_password_protected_guest_link_password_label
+                        ),
+                        value = viewModel.state.password,
+                        placeholderText = stringResource(
+                            id = R.string.conversation_options_create_password_protected_guest_link_button_placeholder_text
+                        ),
+                        onValueChange = viewModel::onPasswordUpdated,
+                        autofillTypes = emptyList()
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+                    WirePasswordTextField(
+                        labelText = stringResource(
+                            id = R.string.conversation_options_create_confirm_password_protected_guest_link_password_label
+                        ),
+                        placeholderText = stringResource(
+                            id = R.string.conversation_options_create_password_protected_guest_link_button_placeholder_text
+                        ),
+                        value = viewModel.state.passwordConfirm,
+                        onValueChange = viewModel::onPasswordConfirmUpdated,
+                        autofillTypes = emptyList()
+                    )
+                    Spacer(modifier = Modifier.height(24.dp))
```

Comment:
question: both of these `WirePasswordTextField` need to be in a single `LazyColumn` item? 🤔